dev/com.ibm.ws.install.featureUtility/src/com/ibm/ws/install/featureUtility/FeatureUtility.java

```diff
@@ -795,8 +770,8 @@ public FeatureUtility build() throws IOException, InstallException {
 
     }
 
-    public static List<String> getMissingArtifactsFromFolder(List<String> artifacts, String location, boolean isShortName) throws IOException, InstallException{
-        List<String> result = new ArrayList<String>();
+    public List<String> getMissingArtifactsFromFolder(List<String> artifacts, String location, boolean isShortName) {
```

Comment:
Did you intend to remove 'static' here? Do we need to check the call heirarchy to see if there are references where others are accessing the method statically?

Reply:
From the call hierchy this method is not called from anywhere. I also did a search on the method name, but this was the only result.

Reply:
got it, if it's not called we probably dont need to stress about it. My only concern is if it's used in CL or somehow externalized.